src/main/java/com/jitterted/mobreg/adapter/in/web/member/MemberLookup.java

```diff
@@ -25,10 +27,14 @@ Member findMemberBy(AuthenticatedPrincipal principal) {
         return memberService.findByGithubUsername(username);
     }
 
+    public List<Member> findAll() {
```

Comment:
MemberLookup is meant exclusively for Security-related actions, e.g., finding by the AuthenticatedPrincipal, so `findAll` shouldn't be here.